### ISSUE TITLE
User-generated scenarios won't run

### ISSUE THREAD

**Issue by SzymonSzott** [DvdMgr/sem#9]
User-generated scenarios from the scratch directory won't run because of a naming mismatch:

```
cp examples/ns-3/examples/wireless/wifi-multi-tos.cc examples/ns-3/scratch/my-multi-tos.cc
sem run --ns-3-path=/home/szott/sem/examples/ns-3 --results-dir=/home/szott/sem/results_my-multi-tos
...
FileNotFoundError: [Errno 2] No such file or directory: '/home/szott/sem/examples/ns-3/build/optimized/scratch/ns3-dev-my-multi-tos-op
```

Comment from Davide:
> It looks like the contents of build/build_status.py are incorrect, and show the wrong path. I am undecided on how to tackle this: we can either modify the wscript to have the build_status.py point to the correct executable or modify the code that looks for the script executable to also take into account the scratch directory.

**Comment by SzymonSzott** [comment]
The error doesn't occur in a stand-alone ns-3-dev repo. After:
```
./waf -d optimized configure; ./waf
cp examples/wireless/wifi-multi-tos.cc scratch/my-multi-tos.cc
./waf --run my-multi-tos
```
the `my-multi-tos` script is executed correctly even though `build/build-status.py` contains:
```
# Programs that are runnable.
ns3_runnable_programs = ['build/scratch/ns3-dev-my-multi-tos-optimized', 'build/scratch/ns3-dev-scratch-simulator-optimized', 'build/scratch/subdir/ns3-dev-subdir-optimized']
```

**Comment by DvdMgr** [comment]
The most straightforward solution to this would be to detect whether the path proposed by build-status contains the scratch folder, and in that case default the script_executable to build/scratch/script-name, since this seems to always be the case for files built from the scratch folder.

I've looked into fixing this at the root by modifying the wscript, but I am not confident with the build system yet to modify that file. I think the solution described above should be good enough!

**Comment by iirzyk** [comment]
I think i did it already, just going to commit changes and u can check whether it suits you!

**Comment by DvdMgr** [comment]
Fixed in #15
